includes/location-functions.php

```diff
@@ -3,54 +3,129 @@
  * Returns markup for the location details.
  *
  * @author RJ Bruneel
- * @since v3.2.8
+ * @since 3.2.8
  * @return string Location details HTML
  */
 function get_location_html() {
 
 	$post = get_post();
 	$post_name = $post->post_name;
+
+	$campus = ( isset( $post->meta['ucf_location_campus'] ) ) ? $post->meta['ucf_location_campus'] : null;
+	$campus = ( $campus ) ? 'Campus: ' . $campus->post_title . '<br>' : "";
+
 	$abbr = isset( $post->meta['ucf_location_abbr'] ) ? $post->meta['ucf_location_abbr'] : null;
+	$abbr = ( $abbr ) ? "Abbreviation: $abbr<br>" : "";
 
 	$id = isset( $post->meta['ucf_location_id'] ) ? $post->meta['ucf_location_id'] : null;
-	$id = ( isset( $id ) && $id . '-' . strtolower( $abbr ) !== $post_name ) ? "Location ID: $id <br>" : "";
-
-	$abbr = ( $abbr ) ? "Abbreviation: " .  $abbr : "";
-	$address = preg_replace( '/,/', '<br>', $post->ucf_location_address, 1 );
-	$lat_lng = $post->meta['ucf_location_lat_lng'];
+	$id = ( isset( $id ) && $id !== $post_name ) ? "Location ID: $id" : "";
+	
+	$lat_lng = ( isset( $post->meta['ucf_location_lat_lng'] ) ) ? $post->meta['ucf_location_lat_lng'] : null;
 	$google_map_link = ( $lat_lng ) ? "https://www.google.com/maps?saddr=&daddr=" . $lat_lng["ucf_location_lat"] . "," . $lat_lng["ucf_location_lng"] : "";
 
 	ob_start();
 ?>
 	<p>
-		<?php echo $id; ?>
+		<?php echo $campus; ?>
 		<?php echo $abbr; ?>
+		<?php echo $id; ?>
 	</p>
 
-<?php if( isset( $address ) ) : ?>
-	<p>
 	<?php if( ! empty( $google_map_link ) ) : ?>
-		<a href="<?php echo $google_map_link; ?>" target="_blank" rel="noopener">
+		<a href="<?php echo $google_map_link; ?>" target="_blank" rel="noopener" class="btn btn-secondary btn-sm text-uppercase mb-4 mt-2">Get Directions</a>
 	<?php endif; ?>
-			<?php echo $address; ?>
-	<?php if( ! empty( $google_map_link ) ) : ?>
-		</a>
-	<?php endif; ?>
-	</p>
-<?php endif; ?>
-
-<?php if( ! empty( $google_map_link ) ) : ?>
-	<a href="<?php echo $google_map_link; ?>" target="_blank" rel="noopener" class="btn btn-secondary btn-sm text-uppercase mb-4 mt-2">Get Directions</a>
-<?php endif; ?>
 
 <?php
 	return ob_get_clean();
 }
 
+/**
+ * Returns a formatted phone link.
+ *
+ * @author RJ Bruneel
+ * @since 3.2.8
+ * @param object $object org object
+ * @param string $index org phone index
+ * @return string Location details HTML
+ */
+function format_phone_link( $object, $index ) {
+	return '<a href="tel:+1' . preg_replace( "/[^0-9,.]/", "", $object[$index] ) . '">' . $object[$index] . '</a>';
+}
+
+/**
+ * Outputs the organization html.
+ *
+ * @author RJ Bruneel
+ * @since 3.2.8
+ * @param string $markup The passed in markup
+ * @param object $obj The page object
+ * @return string The header markup
+*/
+function get_organization_html( $title, $org, $count ) {
```

Comment:
Using and returning an output buffer in this function would make this a little easier to read, and ensure the function actually returns a string like the `@return` docline states